docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -1020,8 +1020,41 @@ public class FooConfiguration {
 
 This replaces the `SpringMvcContract` with `feign.Contract.Default` and adds a `RequestInterceptor` to the collection of `RequestInterceptor`.
 
+`@FeignClient` also can be configured using configuration properties. If we create both `@Configuration` bean and configuration properties, configuration properties will win. It will override `@Configuration` values.
```

Comment:
I am not sure what is right, but my gut tells me that `@Configuration` beans should override properties.